setup.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
+
+from distutils.core import setup
+
+setup(name='demisto',
+      version='1.0',
```

Comment:
shouldn't this be 1.0.1 as well?

Reply:
I'll change (although it doesn't really matter).